packages/lesswrong/lib/collections/users/custom_fields.ts

```diff
@@ -203,6 +203,19 @@ addFieldsDict(Users, {
       }
     },
   },
+  
+  showHideKarmaOption: {
```

Comment:
In this file, the changes are just to the form groups. I also reordered the list a bit to move some things in the same form group together.